lib/src/report/template.dart

```diff
@@ -251,13 +252,59 @@ Future<ReportSection> followsTemplate(PackageContext context) async {
         'See the [example](https://raw.githubusercontent.com/dart-lang/stagehand/master/templates/package-simple/CHANGELOG.md) generated by `stagehand`.',
   );
   final pubspecSection = await checkPubspec();
-  final subsections = [pubspecSection, readmeSubsection, changelogSubsection];
+  final subsections = [
+    pubspecSection,
+    readmeSubsection,
+    changelogSubsection,
+    _licenseSection(await context.licenceTags),
+  ];
   return makeSection(
     id: ReportSectionId.convention,
     title: 'Follow Dart file conventions',
-    maxPoints: 20,
+    maxPoints: 30,
     subsections: subsections,
     basePath: packageDir,
     maxIssues: 10,
   );
 }
+
+Subsection _licenseSection(LicenseTags tags) {
+  const title = 'Uses OSI-approved license';
```

Comment:
To be aligned with the other headings this should be in the imperative:
```suggestion
  const title = 'Use an OSI-approved license';
```